docs/Release_Notes.gfs.v16.3.11.md

```diff
@@ -66,17 +76,22 @@ JOBS CHANGES
 PARM/CONFIG CHANGES
 -------------------
 
-* No changes from GFS v16.3.10
+* parm/wafs/wafs_gcip_gfs.cfg config file was updated
+* Change Externals.cfg to tag = "gfs_wafs.v6.3.2"
```

Comment:
No need to note this here, the new tag is shown above. Remove the `* Change Externals.cfg to tag = "gfs_wafs.v6.3.2"` line.